packages/core/test/unit/segments/segment.test.js

```diff
@@ -430,11 +430,24 @@ describe('Segment', function() {
         segment.flush();
 
         sendStub.should.have.been.calledOnce;
-        var sentSegment = sendStub.lastCall.args;
+        var sentSegment = sendStub.lastCall.args[0];
         assert.notProperty(sentSegment, 'exception');
         assert.notProperty(sentSegment, 'counter');
         assert.notProperty(sentSegment, 'notTraced');
       });
+
```

Comment:
Are you expanding upon one of our tests here? Not sure I understand this change in the context of this PR.

Reply:
Yes this is a test added to ensure prototype properties on segments that are emitted are preserved. See PR body for more detail:

> I've also made a small fix to the tests already present for flush here:  https://github.com/aws/aws-xray-sdk-node/blob/2908ef650c228b51b074063f3d1174afe3b96d42/packages/core/test/unit/segments/segment.test.js#L433
> Noticed that this test wasn't properly functioning when I added my test to explicitly check whether a prototype property was still available on the segment, this because the args is an array of arguments sent to the function, however before this chance the argument array was used as `sentSegment`.

Reply:
I see. Apologies for glossing over that.